WcaOnRails/app/views/persons/_personal_records.html.erb

```diff
@@ -0,0 +1,39 @@
+<div class="personal-records">
+  <h3 class="text-center"><%= t 'persons.show.personal_records' %></h3>
+  <div class="table-responsive">
+    <table class="table table-striped">
+      <thead>
+        <tr>
+          <th class="event"><%= t 'competitions.results_table.event' %></th>
+          <th class="country-rank">NR</th>
+          <th class="continent-rank">CR</th>
+          <th class="world-rank">WR</th>
+          <th class="single"><%= t 'common.single' %></th>
+          <th class="average"><%= t 'common.average' %></th>
+          <th class="world-rank">WR</th>
+          <th class="continent-rank">CR</th>
+          <th class="country-rank">NR</th>
+        </tr>
+      </thead>
+      <tbody>
+        <% @ranks_single.sort_by { |rank_single| rank_single.event.rank } .each do |rank_single| %>
+          <% rank_average = @ranks_average.find { |rank| rank.event_id == rank_single.event_id } %>
+          <tr>
+            <td class="event" data-event="<%= rank_single.event.id %>">
+              <%= cubing_icon rank_single.event.id %>
+              <%= t "events.#{rank_single.event.id}" %>
+            </td>
+            <%= rank_td rank_single, "country" %>
+            <%= rank_td rank_single, "continent" %>
+            <%= rank_td rank_single, "world" %>
+            <td class="single"><%= rank_single %></td>
```

Comment:
In the current PHP code, this links to the work ranking singles for this event. I think in the philisophy of not changing too many things at once, we should probably continue to do so here. Later on, I'm open to discussing changing this.

More importantly, while reading this code, it was pretty surprising to me that this caused the result to get printed out (reading the code, it looks like the persons "rank" would get printed). The reason this works right now is because the `to_s` method for the RanksSingle and RanksAverage classes are written to print out the clock_format of the relevant solve. I'd rather add a `def solve_time` to the `Ranks*` classes and rewrite this line like this:

```ruby
<td class="single"><%= rank_single.solve_time %></td>
```

to make it clear that we're looking at the actual result, not the ranking.